code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm

```diff
@@ -94,6 +99,9 @@
 		calculated_laser_rating += laser.tier
 	min_temperature = max(T0C - (base_cooling + calculated_laser_rating * 15), TCMB) //73.15K with T1 stock parts
 	max_temperature = T20C + (base_heating * calculated_laser_rating) //573.15K with T1 stock parts
+	if(has_anomaly_core)
+		max_temperature += 1500
```

Comment:
Shouldnt the anomaly core overwrite the max temperature? If(core) -> set temp to this, else -> check micro lasers

Reply:
When you are slotting in the core, it is a supplement, not choosing the limit, this allows lasers and upgrades to still have some value as you can raise the temp substantially above 1500 with max lasers

Reply:
I thought the anomaly core was meant to be a replacement to the lasers, being better than t4 and all.

Reply:
The lasers are what allows for cooling because that's what (I assume) allows cooling to be possible so the lasers stay with the anomaly core simply enhancing the one aspect, heating.